framework/global/settings.cpp

```diff
@@ -0,0 +1,225 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "settings.h"
+#include <QSettings>
+
+#include "log.h"
+
+using namespace mu::framework;
+using namespace mu::async;
+
+Settings::Settings()
+{
+#ifndef Q_OS_MAC
+    QSettings::setDefaultFormat(QSettings::IniFormat);
```

Comment:
Is it known why or is this code porting?
Comment would be helpful if the reason is known.

Reply:
This is porting.